gravatar/src/main/java/com/gravatar/services/ProfileService.kt

```diff
@@ -69,8 +71,8 @@ public class ProfileService(private val okHttpClient: OkHttpClient? = null) {
      * @param hash The hash to fetch the profile for
      * @param getProfileListener The listener to notify of the fetch result
      */
-    public fun fetch(hash: Hash, getProfileListener: GravatarListener<UserProfiles>) {
-        fetch(hash.toString(), getProfileListener = getProfileListener)
+    public fun fetchWithListener(hash: Hash, getProfileListener: GravatarListener<UserProfiles>) {
```

Comment:
I think we can remove the `WithListener` from the function names; it looks cleaner, but I do not have a strong opinion.

Reply:
I want to keep these public obviously but I don't promote their usage, I feel like keeping the same name might not help with that.